channels/serializers/posts.py

```diff
@@ -101,15 +101,15 @@ def get_created(self, instance):
 
     def get_channel_name(self, instance):
         """The channel which contains the post"""
-        return instance.subreddit.display_name
+        return instance.channel.name or instance.subreddit.display_name
```

Comment:
`channel.name` would only be falsey if we didn't run the backpopulate yet, right?

Reply:
Correct, it _might_ be possible to get an empty string in there somehow, but i'm pretty sure the UI at least doesn't allow it.

Reply:
I looked into adding placeholder text to the prosemirror input and it looks nontrivial. Do you mind if we skip it for now and add it as a separate issue?

Reply:
Whoops wrong issue